analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd

```diff
@@ -0,0 +1,386 @@
+```
+## How many samples have paired genomic and transcriptomic assays?
+There are `r kids_pbta_samples_paired` patient cases with paired genomic and transcriptomic assays.
+
+## All time points
+```{r}
+paired_assays %>%
+  group_by(descriptors) %>%
+  tally() %>%
+  arrange(desc(n)) %>% 
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+```
+
+```{r}
+paired_assays %>%
+  group_by(Kids_First_Participant_ID, descriptors, experimental_strategy) %>%
+  tally() %>%
+  arrange(desc(n)) %>%
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+```
+
+```{r}
+paired_assays_matched <- paired_assays %>%
+  filter(descriptors %in% c("Diagnosis, Recurrence", "Diagnosis, Progressive", "Deceased, Diagnosis", 
+                            "Recurrence, Second Malignancy", "Progressive, Recurrence", "Diagnosis, Second Malignancy", 
+                            "Deceased, Progressive, Recurrence", "Diagnosis, Progressive, Recurrence", "Deceased, Recurrence", "Deceased, Diagnosis, Recurrence",
+                            "Diagnosis, Unavailable")) %>%
+  write_tsv(file.path(results_dir, "paired_assays_matched_time_points.tsv"))
+
+paired_assays_matched %>%
+  group_by(Kids_First_Participant_ID, descriptors, experimental_strategy) %>%
+  tally() %>%
+  arrange(desc(n)) %>%
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+
+matched_samples_paired <- length(unique(paired_assays_matched$Kids_First_Participant_ID))
+```
+## How many samples have paired genomic and transcriptomic assays and matched longitudinal information?
+There are `r matched_samples_paired` patient cases with matched longitudinal samples for genomic and transcriptomic assays.
+
+# Genomic assays and cancer_group
+Let us investigate the number of tumor samples with genomic assays per cancer_group.
+```{r}
+list <- genomic_assays %>%
+  count(cancer_group) %>%
+  arrange(desc(n))
+print(list, dplyr.print_min = Inf)
+```
+
+We will look into the first five cancer groups with the highest number of tumor samples: Low-grade glioma, Medulloblastoma, Diffuse midline glioma, High-grade glioma, and Ependymoma. There are cases that the second time point has a different cancer type. Let us summarize this to ensure we capture all tumor samples regardless of diagnosis changes at a later stage of the disease.We will only do this for the first five histologies for now.
+
+```{r}
+genomic_assays_matched <- genomic_assays_matched %>% 
+  mutate(cancer_group_broad = case_when(grepl("Low-grade glioma", cancer_group) ~ "Low-grade glioma",
+                                        grepl("Medulloblastoma", cancer_group) ~ "Medulloblastoma", 
+                                        grepl("High-grade glioma", cancer_group) ~ "High-grade glioma",
+                                        grepl("Ependymoma", cancer_group) ~ "Ependymoma", 
+                                        grepl("Diffuse midline glioma", cancer_group) ~ "Diffuse midline glioma"))
+```
```

Comment:
Good solution here for resolving the issue I just mentioned :). I did notice that there is one patient with `cancer_group == "High-grade glioma, Low-grade glioma"`, and as of right now that would be assigned to the LGG group. It might be better to prioritize HGG assignment over LGG for cases like this.

Reply:
Great catch! I didn't realize that would have been replaced. I switched the order, see here: 0704391c08132a297251dc30eaa9b51ef958e3ab